lib/rules/comma-dangle.js

```diff
@@ -12,7 +12,9 @@
 //------------------------------------------------------------------------------
 
 module.exports = function (context) {
-    var forbidDanglingComma = context.options[0] !== "always";
+    var allowDangle = context.options[0] || "never";
```

Comment:
I'm not sure what the ESLint convention is here, but wouldn't it make sense to throw for unrecognized options? Throwing errors makes it easier for the user to spot errors such as typos.